sdk/python/kfp/compiler/_component_builder.py

```diff
@@ -58,6 +59,78 @@ def download_gcs_blob(local_path, gcs_path):
     blob = bucket.blob(gcs_blob)
     blob.download_to_filename(local_path)
 
+class DependencyVersion(object):
```

Comment:
What do you think about something like `VersionedDependency` which also contains the package name? Why leave the name outside?

Reply:
That is a great idea, I included the name in the VersionedDependency.